### PR TITLE
Cleaned up spec and small code change

### PR DESCRIPTION
**Pull request by danieldobalian**
Updated to comment out ellipsis, fixed small iOS bug, and switched order of a few bullets for clarity.

### CODE REVIEW

**Comment by danieldobalian** [comment]
@dstrockis

**Comment by azurecla** [comment]
Hi __@danieldobalian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by PRmerger** [comment]
@danieldobalian : Thanks for your contribution to the Azure documentation! The author, @dstrockis, has been notified to review your proposed change.

File: articles/active-directory/active-directory-devquickstarts-xamarin.md [hunk 1]
```diff
@@ -114,37 +114,37 @@ public static async Task<List<User>> SearchByAlias(string alias, IPlatformParame
...
 ```
 
 That's all for the `DirectorySearcher` PCL and your app's identity-related code.  All that remains is to call the `SearchByAlias(...)` method in each platform's views, and where necessary add code for correctly handling the UI lifecycle.
 
 ####Android:
-- In `MainActivity.cs`, add a call to `SearchByAlias(...)` in the button click handler:
-
-```C#
-List<User> results = await DirectorySearcher.SearchByAlias(searchTermText.Text, new PlatformParameters(this));
-```
```
Comment (dstrockis) [inline articles/active-directory/active-directory-devquickstarts-xamarin.md]: @danieldobalian why remove these lines?

File: articles/active-directory/active-directory-devquickstarts-xamarin.md [hunk 2]
```diff
@@ -162,16 +162,16 @@ List<User> results = await DirectorySearcher.SearchByAlias(
 ```C#
 List<User> results = await DirectorySearcher.SearchByAlias(
   SearchTermText.Text,
-  new PlatformParameters(PromptBehavior.Auto, this.Handle));
+  new PlatformParameters(this));
 ```
```
Comment (dstrockis) [inline articles/active-directory/active-directory-devquickstarts-xamarin.md]: @danieldobalian is this right? I recall having to pass a handle to the window on Windows Desktop. Did you make the corresponding change in the quickstart code?

Comment (danieldobalian) [inline articles/active-directory/active-directory-devquickstarts-xamarin.md]: This was meant for iOS.

File: articles/active-directory/active-directory-devquickstarts-xamarin.md (hunk 1 above)
Comment (danieldobalian) [inline articles/active-directory/active-directory-devquickstarts-xamarin.md]: Not intentional.

**Pull request closed without merging**